CLI/FileUtils.cpp

```diff
@@ -244,8 +247,11 @@ std::vector<std::string> getSourceFiles(int argc, char** argv)
 
     for (int i = 1; i < argc; ++i)
     {
-        if (argv[i][0] == '-')
+        // Treat '-' as a special file whose source is read from stdin
+        // All other arguments that start with '-' are skipped
+        if (argv[i][0] == '-' && argv[i][1] != '\0') {
```

Comment:
Inconsistent bracing style, no need for braces here though.

Reply:
Done